requirements.txt

```diff
@@ -1,8 +1,7 @@
-git+https://github.com/vmenger/deduce@fast-temp#egg=deduce
+deduce==2.0.0
 flask_restx==0.5.1
 pytest==6.2.2
```

Comment:
Ik kreeg een error met Python 3.10 en deze versie van PyTest. Met een conda upgrade kreeg ik versie 7.2.0, en die doet het wel.

https://stackoverflow.com/questions/69528842/an-error-while-trying-to-execute-tests-on-python-3-10-with-pytest